### DISCUSSION TITLE
Copilot Bugs  In vs code

### DISCUSSION THREAD

**Discussion by soummyaanon** [community/community#110582] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

10 Out of 3 times Whenever I ask a Question it gives  404 Error even after The Internet connection is There  so what is the solution

**Comment by davecheney** [comment, marked as answer]
@soummyaanon it is likely you are running an old version of the chat extension. Please see https://github.com/orgs/community/discussions/110196
